css/types/calc.json

```
@@ -222,6 +222,38 @@
               "deprecated": false
             }
           }
+        },
+        "typed_arithmetic": {
+          "__compat": {
+            "description": "Typed arithmetic",
```

Let's prefer the spec term:

```suggestion
        "type_checking": {
          "__compat": {
            "description": "Type checking",
```

---

The trouble with this is that:

a. "Typed arithmetic" is the common name being used in the community to describe this change. If we call it "Type checking", no one will know what we are talking about. "Type checking (typed arithmetic)" might be a viable alternative.
b. Strictly speaking, the actual change that had occurred is a change in the behavior of type checking (it existed before this change), which means it is now valid to divide two typed values (which can be different units, but they must be the same overall type) to produce a unitless value.

But this is quite difficult to summarize succinctly ;-)

---

So, if I read the css-tricks article correctly, there is no new "type checking" going on here. Instead, I would summarize the new feature we're talking about here as "typed_division_produces_unitless_number". Would you agree?